icons/logos/google.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+  <path d="M21.8,10h-2.6l0,0H12v4h5.7c-0.8,2.3-3,4-5.7,4c-3.3,0-6-2.7-6-6s2.7-6,6-6c1.7,0,3.2,0.7,4.2,1.8l2.8-2.8C17.3,3.1,14.8,2,12,2C6.5,2,2,6.5,2,12s4.5,10,10,10s10-4.5,10-10C22,11.3,21.9,10.6,21.8,10z"/>
+</svg>
```

Comment:
It looks like the formatting of this SVG file is slightly different than the rest of the files. Could you run `npm run build` to fix it? There are instructions on how to do that at the bottom of this comment: https://github.com/feathericons/feather/issues/171#issuecomment-455356985

Reply:
Fixed!